contracts/modules/SSVOperators.sol

```diff
@@ -62,6 +62,7 @@ contract SSVOperators is ISSVOperators {
         operator.snapshot.balance = 0;
         operator.validatorCount = 0;
         operator.fee = 0;
+        operator.whitelisted = false;
```

Comment:
Just so i understand it all, this is the main thing in the PR right?
And the only implication of this was that in the getOperatorById view command it would be inaccurate for whitelisted true but removed operator right?